<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TmachineDetailSetupViewItemTable</class>
 <widget class="QWidget" name="TmachineDetailSetupViewItemTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1001</width>
     <height>701</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QTableWidget" name="m_machineGroupFuncBaseSetupTable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>120</number>
      </attribute>
      <column>
       <property name="text">
        <string>出场厂家</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>厂家电话</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>存放地点</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>营业状况</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>备注</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
